index.html

```diff
@@ -341,35 +341,45 @@ <h1>Shareholders of MTL and affiliates</h1>
         }
       }
 
-      async function getDelegations(asset) {
-        const response =
-          await
-            fetch(
-              'https://api.stellar.expert/explorer/public/payments?' +
-              `asset=${asset.code}-${asset.issuer}&limit=200&order=desc`
-            );
-        const responseJson = await response.json();
-        const payments = responseJson._embedded.records;
-
-        let sources = {};
-        let targets = {};
-        for (const payment of payments) {
-          if (payment.from in sources)
-            continue; // keep only first met (last happened) delegation
-          switch (payment.memo) {
-            case 'delegate':
-              sources[payment.from] = payment;
-              if (!(targets[payment.to]))
-                targets[payment.to] = {};
-              targets[payment.to][payment.from] = payment;
-              break;
-            case 'undelegate':
-              sources[payment.from] = false;
-              break;
-          }
-        }
+      async function getDelegations(asset)
+      {
+        let currentData :String = localStorage.getItem(asset);
```

Comment:
The request type and asset code must be included in the key. Other parts of the asset are unnecessary.